README.md

```diff
@@ -5,7 +5,8 @@ A HTML5 fullscreen plugin for Leaflet.
 
 ``` js
 var map = new L.Map('map', {
-    fullscreenControl: true
+    fullscreenControl: true,
+    fullscreenPseudo: false // if true, fullscreen to page width and height
```

Comment:
Ok

But how set pseudoFullscreen option if i want use custom control button (not from fullscreen plugin) and pseudoFullscreen mode?